src/components/composer/composer/composer.ts

```diff
@@ -81,6 +80,10 @@ css/* scss */ `
   .o-topbar-toolbar .o-composer-container:focus-within {
     border: 1px solid ${SELECTION_BORDER_COLOR};
   }
+
+  .o-topbar-toolbar .o-composer-container {
+    z-index: ${ComponentsImportance.TopBarComposer};
```

Comment:
> +1 Just watch out, starting 16.2 (I think) we have a dedicated component `TopBarComposer`, it will be the perfect occasion to move that rule to the component :)

This has already been done and the issues in saas-16.2 and above are already resolved.

Reply:
ok, this means I can stop the forward port to saas-16.1, right ?

Reply:
> ok, this means I can stop the forward port to saas-16.1, right ?

Yes, let it go until saas-16.1. It will create a new PR with all necessary changes for saas.16.2 and above